### ISSUE TITLE
cs_config.py: config.compilers['version'] = @CC_VERSION@ not replaced by autotools

### ISSUE THREAD

**Issue by christoph-conrads** [code-saturne/code_saturne#83]
In the code below, `@CC_VERSION@` is not replaced by autotools when building Code_Saturne 6.0.5, 6.3.1, and 7.0.0 in a Linux container with Devuan Beowulf (Debian 10 without systemd).

Template input:
https://github.com/code-saturne/code_saturne/blob/3ced548a007633199a3ae1c4f63ececdb64ddee9/bin/cs_config.py.in#L104-L108

Output (here with `prefix.cs-6.3.1/lib/python3.7/site-packages/code_saturne/cs_config.py`):
```python
        self.compilers = {'cc': "gcc",
                          'cxx': "g++",
                          'fc': "gfortran",
                          'ld': "gcc",
                          'version': "@CC_VERSION@"}
```

Let me know if you need log files.

**Comment by christoph-conrads** [comment]
Line 171 is also not substituted by Autotools:
https://github.com/code-saturne/code_saturne/blob/3ced548a007633199a3ae1c4f63ececdb64ddee9/bin/cs_config.py.in#L171

**Comment by YvanFournier** [comment]
The CC_VERSION issue has been fixed in the master branch in commit 4a08ac54.

The USE_NLS issue is related to the removal/deactivation of gettext-based internationalization in v6.1, so I also need to remove that.

No need for log files here, I reproduce this.

**Comment by YvanFournier** [comment]
Fixed in commit c55950be (master).
